src/API/Site/Controllers/Ads/AssetGroupController.php

```diff
@@ -42,6 +43,17 @@ public function __construct( RESTServer $rest_server, AdsAssetGroup $ads_asset_g
 	 * Register rest routes with WordPress.
 	 */
 	public function register_routes(): void {
+		$this->register_route(
+			'ads/campaigns/asset-groups/(?P<id>[\d]+)',
```

Comment:
❓   Why here in the PUT the ID param is at the end and in the GET is before the asset-groups URL segment?

Reply:
In the `PUT` the `id` is the `asset group id` and in the `GET` the `id` is the `campaign id`, however, I am considering renaming the `GET` endpoint ( in a different PR), so instead of using  `ads/campaigns/CAMPAIGN_ID/asset-groups` I think it is more convenient something like: `ads/campaigns/asset-groups?campaign_id=123445`, WDYT?

Reply:
Ahhh gotcha. I missed that. For me both options are valid tho.